### PR TITLE
Added wiring for PHP Application V2 Snippets

### PR DESCRIPTION
**Pull request by dragonmantank**
## Description

Added the PHP Application V2 Code Snippets

## Deploy Notes

N/A

### CODE REVIEW

**Comment by tbedford** [comment]
Probably needs to point to updated submodule.

**Comment by mheap** [comment]
```
CodeSnippetFilter - Could not load /home/travis/build/Nexmo/nexmo-developer/.repos/nexmo/nexmo-php-code-snippets/applications/create-application.php for language php
```

Exactly that, Tony!

@dragonmantank The submodule was merged in to master. Can you rebase this branch on to `master` to fix the build?

File: _examples/application/create-application/php.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: PHP
```
Comment (mheap) [inline _examples/application/create-application/php.yml]: The only parts we need in these files are `client` and `code`. The rest all uses smart defaults now (and we should clean up the old snippets)

Comment (dragonmantank) [reply _examples/application/create-application/php.yml]: Removed those chunks from all the pages, and rebased on current master.

**Comment by dragonmantank** [comment]
@mheap Rebased on newest master with the refined code snippets, and renumbered since we removed some lines from there.

**Pull request merged**